cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -712,9 +740,11 @@ impl pallet_aura::Config for Runtime {
 	type AuthorityId = AuraId;
 	type DisabledValidators = ();
 	type MaxAuthorities = ConstU32<100_000>;
-	type AllowMultipleBlocksPerSlot = ConstBool<false>;
+	// TODO: should this be true? Guide says false, but see:
+	// https://github.com/paritytech/polkadot-sdk/pull/1619/files#diff-17c0e44911184254502a1d0b157c58e706eaa938cd0b1fe34c0456e541066a2fR234
+	type AllowMultipleBlocksPerSlot = ConstBool<true>;
```

Comment:
Yeah interesting, `false` makes more sense to me. @eskimor ?

Actually, guide says `true`: https://github.com/w3f/polkadot-wiki/blob/master/docs/maintain/maintain-guides-async-backing.md#phase-3---activate-async-backing

Reply:
Yeah, I figure that's the intended final value. It's just a bit confusing because we set it to `false` before that, in step 1.5.